service/watcher/chartstatus/dynclient.go

```diff
@@ -116,21 +109,29 @@ func (c *ChartStatusWatcher) waitForChartOperator(ctx context.Context) (*v1alpha
 			return microerror.Mask(err)
 		}
 
+		err = c.k8sClient.CtrlClient().Get(
+			ctx,
+			types.NamespacedName{
+				Name:      key.KubeConfigSecretName(chartOperatorAppCR),
+				Namespace: key.KubeConfigSecretNamespace(chartOperatorAppCR),
+			},
+			&kubeConfigSecret,
+		)
+		if err != nil {
+			return microerror.Mask(err)
+		}
```

Comment:
Bug was because we were waiting for the chart-operator app CR to be created but not checking its kubeconfig secret exists.

This causes a large volume of error logs until the kubeconfig secret has been created.